src/App.js

```diff
@@ -5,6 +5,13 @@ import './App.scss';
 import posts from './api/posts';
 import comments from './api/comments';
 import users from './api/users';
+import { Postlist } from './postlist/Postlist';
+
+const database = posts.map(post => ({
```

Comment:
database is a bit confusing name for a variable

```suggestion
const preparedPosts = posts.map(post => ({
```